dev/Controller/SPController.php

```diff
@@ -140,12 +120,8 @@ public function assertionConsumerServiceAction(Request $request)
 
     /**
      * Formats xml.
-     *
-     * @param string $xml
-     *
-     * @return string
      */
-    private function toFormattedXml($xml)
+    private function toFormattedXml(string|bool $xml): string|bool
```

Comment:
Kan hier $xml de waarde true/false hebben?

Reply:
Yes it can, we directly pass the contents of the `base64_decode` function into this method. Im going to add a guard to prevent this. And then the type hint can be more strict! Good point mate!